gcloud/orb.yaml

```diff
@@ -247,9 +247,9 @@ jobs:
         default: small
         type: string
       memory:
+        default: ''
```

Comment:
Note: this makes it a breaking change, please be sure to deploy as a new major version.